standard.md

```diff
@@ -4,19 +4,19 @@
+**3. Clear Ownership** | Ownership of everything the project produces must be clearly defined and documented. For example, through copyright, trademark or other publicly available information.
+**4. Platform Independence** | If the project has mandatory dependencies that create more restrictions than the original license, the project(s) must be able to demonstrate independence from the closed component(s) and/or indicate the existence of functional, open alternatives.
+**5. Documentation** | The project must have documentation of the source code, use cases, and/or functional requirements. For content, this should include all relevant/compatible apps, software, or hardware required to access the content, and instructions regarding how to use it. For software projects, this should be technical documentation that would allow a technical person unfamiliar with the project to launch and run the software. For data projects, this should be documentation that describes all the fields in the set, and provides context on how data was collected, and how it should be interpreted.
+**6. Mechanism for Extracting Data** | If the project has non personally identifiable information (PII) there must be a mechanism for extracting or importing non PII-data from the system in a non-proprietary format.
+**7. Adherence to Privacy and Applicable Laws** | The project must state to the best of its knowledge that it complies with relevant privacy laws, and all applicable international and domestic laws.
+**8. Adherence to Standards & Best Practices** | Projects must demonstrate adherence to standards, best practices, and/or principles. For example, [the Principles for Digital Development](https://digitalprinciples.org/principles/).
+**9. Do No Harm** | All projects must demonstrate that they have taken steps to ensure the project anticipates, prevents, and does no harm.
+**9a) Data Privacy & Security** | Projects that collect data must identify the types of data collected and stored and demonstrate that the project ensures the privacy and security of this data and has taken steps to prevent adverse impacts resulting from it’s collection, storage and distribution.
```

Comment:
This is an awfully long sentence with 5 `and` clauses. Kindly break it down in shorter sentences.

Reply:
Agreed - just changed to make it easier to follow/read.